.github/workflows/ruby_tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.5
+          ruby-version: 2.7
```

Comment:
wdyt about using ruby `2.7` also for tests? (and drop `2.5` and `2.6`)

Reply:
Ruby 2.5 is still used on Debian 10 and Ubuntu 18.04 and both are supported platforms.